src/styles/typography.css

```diff
@@ -250,3 +257,57 @@ h1, h2, h3, h4, h5, h6, p, div, span, section, li {
 .bold:not([class*="heading-"]) {
   font-weight: var(--body-bold-font-weight);
 }
+
+/*
+ * =========================
+ * Text weights
+ * =========================
+ */
+
+.text-extra-light {
+  font-weight: --text-weight-extra-light;
+}
+
+.text-light {
+  font-weight: --text-weight-light;
+}
+
+.text-regular {
+  font-weight: --text-weight-regular;
+}
+
+/*
+ * while lighter weights get important readability/clarity benefits
+ * from subpixel antialiasing, heavier weights lose clarity in areas
+ * like the inside of loops/curves etc. and can appear much darker than
+ * intended with the same color as a lighter weight. Let's move up to
+ * the full pixel antialiasing when we hit semibold, and rely on text color
+ * to ensure contrast ratios are still sufficiently high.
+ */
```

Comment:
fyi @gcomstock — I made this (hopefully not controversial?) judgement call while I was fleshing out some history view stuff based on observations about how our various text weights/colors were working in practice vs. the mockups. Happy to discuss more and/or take a different path if you disagree.

Reply:
I'll take a look on your machine, but I trust your eye on this! Plenty of ways to establish visual hierarchy while keeping text readable.